istio/oam/register.go

```diff
@@ -117,3 +118,11 @@ func load(basePath string) ([]schemaDefinitionPathSet, error) {
 
 	return res, nil
 }
+func init() {
+	var err error
+	pathSets, err = load(workloadPath)
+	if err != nil {
+		fmt.Printf("Could not load definitions and schemas for static component registeration: %v", err.Error())
```

Comment:
Can this be a MeshKit error?

Reply:
load returns a filepath type error, so I didn't use meshkit logger. And this package didn't have the errors.go already, so I didn't create a meshkit error. Will do now

Reply:
Also this happens before main function is run, so we dont have the meshkit logger instance yet. I can use normal logger for meshkit error, but then it is of no use